features/nanostack/nanostack-interface/mbed_lib.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "nanostack-interface"
```

Comment:
Can this be part of the "adaptation" package..
I don't see the point of nanostack-hal-mbed-cmsis-rtos and nanostack-interface and mbed-mesh-api to be split to so many "modules" as they are basically just one adaptation layer between Nanostack and Mbed OS.

Reply:
this one i can fix